.buildkite/pipeline.yml

```diff
@@ -53,16 +61,21 @@ steps:
   #################
   # Run Unit Tests
   #################
-  - label: "🔬 Unit Tests"
-    command: ".buildkite/commands/run-unit-tests.sh"
-    depends_on: "build"
-    env: *common_env
-    plugins: *common_plugins
-    artifact_paths:
-      - "build/results/*"
-    notify:
-      - github_commit_status:
-          context: "Unit Tests"
+  - group: "🔬 Unit Tests"
+    steps:
+      - label: "🔬 Run Unit Tests"
+        command: ".buildkite/commands/run-unit-tests.sh"
+        depends_on: "build"
+        env: *common_env
+        plugins: *common_plugins
+        artifact_paths:
+          - "build/results/*"
+        notify:
+          - github_commit_status:
+              context: "Unit Tests"
+      - wait: ~
+        continue_on_failure: true
+      - *annotate_test_failures_step
```

Comment:
I just realized that I'll need to provide an explicit and different `context:` key for each of the 3 invocations of this plugin, so that it creates 3 different annotations when the Unit Test, iPhone UI Tests and iPad UI Tests all fail, otherwise each run will override the annotation of the previous one.

So we won't be able to keep that pipeline as neat after all, and might have to copy/paste the step after all just to specify an explicit and different `context:` for each.